src/entities/course/ui/course-card/course-card.test.tsx

```diff
@@ -21,20 +22,24 @@ describe('CourseCard', () => {
     renderWithRouter(<CourseCard {...mockProps} />);
   });
 
+  it('renders the course card correctly', () => {
+    expect(screen.getByTestId('course-card')).toBeInTheDocument();
+  });
+
   it('renders the course card title', () => {
-    expect(screen.getByText(mockProps.title)).toBeInTheDocument();
+    expect(screen.getByText(mockProps.title)).toBeVisible();
   });
 
   it('renders the course start date', () => {
-    expect(screen.getByText(`${mockProps.startDate}`)).toBeInTheDocument();
+    expect(screen.getByText(`${mockProps.startDate}`)).toBeVisible();
   });
 
   it('renders the course language', () => {
-    expect(screen.getByText(mockProps.language.join(' / '))).toBeInTheDocument();
+    expect(screen.getByText(mockProps.language.join(' / '))).toBeVisible();
   });
 
   it('renders the course mode', () => {
-    expect(screen.getByText(`${mockProps.mode}`)).toBeInTheDocument();
+    expect(screen.getByText(`${mockProps.mode}`)).toBeVisible();
   });
```

Comment:
It would be good to discuss with the team whether we should separate the tests like this or whether we should combine them